corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -94,7 +94,7 @@ def testBasicRebuild(self):
         case.xform_ids = [case.xform_ids[0], case.xform_ids[2], case.xform_ids[1]]
         case.rebuild()
         self.assertEqual(case.p1, 'p1-1') # original
-        self.assertEqual(case.p2, 'p2-1') # updated (back!)
+        self.assertEqual(case.p2, 'p2-2') # updated (back!)
```

Comment:
i'm confused by this - I think the test was intending to test that if you flipped the action order then `p2-1` would win out over `p2-2`. I think what may be happening now is that the rebuild process is also sorting the forms which now have granular enough dates that flipping the action order gets reverted during the sorting that happens in rebuild. so this test is correct but no longer testing the thing it was originally intended to test.

can we just set all the action dates to be the same here? or override the datetime format?